Documentation/guideline/csharp_coding_standards.md

```diff
@@ -112,24 +118,24 @@ The class definition contains class members in the following order, from *less*
 ### Cross-platform coding
 Our code should supports multiple operating systems. Don't assume we only run (and develop) on Windows. Code should be sensitvie to the differences between OS's. Here are some specifics to consider.
 
-* **DO** use `Enviroment.NewLine` instead of hard-coding the line break instead of `\r\n`, as Windows uses `\r\n` and OSX/Linux uses `\n`.
+* **DO** use `Enviroment.NewLine` instead of hard-coding the line break, as Windows uses `\r\n` and OSX/Linux uses `\n`.
 
-> **Note**
->
-> Be aware that thes line-endings may cause problems in code when using `@""` text blocks with line breaks.
+  > **Note**
+  >
+  > Be aware that thes line-endings may cause problems in code when using `@""` text blocks with line breaks (it spans over multiple lines).
```

Comment:
The following is exactly the one that may cause problems, as under Windows, `x` may equal to `"line1\r\nline2"`. However I feel "spans over multiple lines" does not point out the issue explictly.
```
var x = @"line1
line2";

Assert.AreEqual("line1\nline2", x);
```